src/pages/Content/index.js

```diff
@@ -11,8 +11,8 @@ chrome.runtime.onMessage.addListener(function (msgObj) {
       sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
       break;
     case 'startPlaying':
-      console.log('Should start playing recording...');
-      sendMessage({ action: 'test', value: 'Hi from content (player)!' });
+      console.log('Should start playing recording...', recordedEvents.length);
```

Comment:
Add better formatting and explanation of the `.length`.

Reply:
I will remove this line as it was added there to test during the development.

Reply:
removed it.